src/Microsoft.Azure.SignalR/ServerConnections/ServiceConnection.cs

```diff
@@ -111,9 +111,29 @@ protected override Task DisposeConnection(ConnectionContext connection)
         return _connectionFactory.DisposeAsync(connection);
     }
 
-    public override Task CloseClientConnections(CancellationToken token)
+    public override async Task CloseClientConnections(CancellationToken token)
```

Comment:
Will this similar cleanup interrupt with below function `CleanupClientConnections`?

Reply:
no, close client connections only do close.